_dev/js/back/src/store/modules/paypal/actions.js

```diff
@@ -46,6 +46,19 @@ export default {
       return Promise.resolve(response);
     });
   },
+  updatePaypalStatusSettings({ commit, getters }) {
+    return ajax({
+      url: getters.adminController,
+      action: 'LiveStepConfirmed'
+    }).then(resp => {
+      if (resp) {
+        commit(types.UPDATE_CONFIRMED_LIVE_STEP);
+        return Promise.resolve(true);
+      }
+
+      return Promise.reject(resp);
```

Comment:
Same here

`throw resp` instead `new Promise.reject(resp)`

Constructing Promises when are unnecesary is considered somehow an antipattern

Reply:
done in the new commit